### ISSUE TITLE
Commit msg fails to update correctly after commits

### ISSUE THREAD

**Issue by sora-blue** [desktop/desktop#16546]
### The problem

After I made one commit, it seemed to be repeating the first msg, 
and my new commit was nowhere to find. It would need a restart 
to show git commit logs correctly. 
Here's the screenshot 
after changes are commited and pushed to the origin:
![image](https://user-images.githubusercontent.com/56759083/232796571-a43f4fad-f028-4b28-943e-1fad60f8ffa3.png)
( I cannot activate the mysterious commit msg at the bottom )
While the online commit logs are:
![image](https://user-images.githubusercontent.com/56759083/232797239-fa7fb459-d3ab-44c9-9709-bf541a6fb442.png)

### Release version

Version 3.2.1 (x64)

### Operating system

Windows 11 22H2

### Steps to reproduce the behavior

0. push local repo to GitHub and  work on it for a while, close & restart
1. commit changes with Github Desktop
2. see commit logs

### Log files

[2023-04-18.desktop.production.log](https://github.com/desktop/desktop/files/11262480/2023-04-18.desktop.production.log)

### Additional context

have checked for updates till this issue

**Comment by sora-blue** [comment]
Yet minutes later the commit log turned out to be fine, and another commit did not have the problem, i don't why
![image](https://user-images.githubusercontent.com/56759083/232803142-c8119fb5-37a7-4ad9-a95d-5b96c2a850b0.png)

**Comment by steveward** [comment]
@sora-blue thanks for the report. I feel like I've seen another report of this previously, but I can't find it at this moment. I'll close this out since it was resolved, but let us know if you run into this again.

**Comment by sora-blue** [comment]
@steveward Glad to hear response! 
Today I got the same situation again, right after one commit was added with GitHub Desktop.  Hope it will be fixed in later versions.
![image](https://user-images.githubusercontent.com/56759083/233002777-7ea954ff-a7bf-4676-ba6f-5632282e97fc.png)
Online logs are good though:
![image](https://user-images.githubusercontent.com/56759083/233003000-5125ae97-a9ee-494f-933b-dd58cfd28ac8.png)
